aspnetcore/signalr/hub-filters.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Use hub filters in SignalR for ASP.NET Core
+author: brecon
+description: Learn how to use hub filters in ASP.NET Core SignalR.
+monikerRange: '>= aspnetcore-5.0'
+ms.author: brecon
+ms.custom: mvc
+ms.date: 05/22/2020
+no-loc: [Blazor, "Identity", "Let's Encrypt", Razor, SignalR]
+uid: signalr/hub-filters
+---
+
+# Use hub filters in SignalR for ASP.NET Core
+
+Hub filters allow logic to run before and after hub methods are invoked by clients. This article will provide guidance for writing and using hub filters.
+
+## Configure hub filters
+
+Hub filters can be applied globally or per hub type. When adding filters the order the filters are added is the order that the filters will run in. Also, global hub filters will run before local hub filters.
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddSignalR(hubOptions =>
+    {
+        // Global filters will run first
+        hubOptions.AddFilter<CustomFilter>();
+    }).AddHubOptions<MyHub>(options =>
```

Comment:
Any particular reason for using hubOptions above and options here?

Reply:
Copied from somewhere... but I'll change it